src/components/ui/Pool/PoolZapOutModal.tsx

```
@@ -0,0 +1,218 @@
+            await approveToken(
+                CONTRACT_ADDRESSES.weweVault, //we need to approve SHARES token
+                CONTRACT_ADDRESSES.zapContract,
+                ethers.parseUnits(data.zapOutAmount, 18) //shares decimals, we SHOULD NOT HARDCODE THEM
+            )
+        const txReceipt = await zapOut(
+            selectedPool?.address, 
+            zapOutToken!.address,  
+            ethers.parseUnits(data?.zapOutAmount, 18) //shares decimals, we SHOULD NOT HARDCODE THEM
+            .toString() as `0x${string}`)
+
+            const totalFee = (txReceipt!?.gasUsed * txReceipt!?.gasPrice);
```

Biome

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)